pandas/tests/series/test_missing.py

```diff
@@ -891,6 +891,16 @@ def test_spline_error(self):
         with tm.assertRaises(ValueError):
             s.interpolate(method='spline', order=0)
 
+    def test_interp_timedelta64(self):
```

Comment:
sure, would it be better if i add it as one more in the existing test or as a new test?